src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -211,8 +211,10 @@
   nonstatic_field(ArrayKlass,                  _dimension,                                    int)                                   \
   volatile_nonstatic_field(ArrayKlass,         _higher_dimension,                             ObjArrayKlass*)                        \
   volatile_nonstatic_field(ArrayKlass,         _lower_dimension,                              ArrayKlass*)                           \
-  nonstatic_field(CompiledICHolder,            _holder_metadata,                              Metadata*)                             \
-  nonstatic_field(CompiledICHolder,            _holder_klass,                                 Klass*)                                \
+  volatile_nonstatic_field(CompiledICData,     _speculated_method,                            Method*)                               \
+  volatile_nonstatic_field(CompiledICData,     _speculated_klass,                             uintptr_t)                             \
+  nonstatic_field(CompiledICData,              _itable_defc_klass,                            Klass*)                                \
+  nonstatic_field(CompiledICData,              _itable_refc_klass,                            Klass*)                                \
```

Comment:
I don't think it makes sense to export these fields and types until SA is changed to make use of them.

Reply:
Good point. I thought at least Graal needs them, but they have their own VM structs file where they are exposed. I'll remove this.